draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -1027,7 +1030,7 @@ It should also be possible to correlate information from IP and
 optical layers (e.g., which port, lambda/OTSi, and direction are used
 by a specific IP service on the WDM equipment).
 
-In particular, for the cross-layer links, it is key for MDSC to
+In particular, for the Cross-technology links, it is key for MDSC to
```

Comment:
Cross-technology **Ethernet** links should be used everywhere in the document according to the terminology definition made at the beginning

Reply:
Done in https://github.com/IETF-TEAS-WG/actn-poi/pull/118/commits/ee540dfe2e2edce459a4dd672956804b2c11b01b